src/composables/queries/usePoolSnapshotsQuery.ts

```diff
@@ -52,9 +52,16 @@ export default function usePoolSnapshotsQuery(
     let prices: HistoricalPrices = {};
 
     const isStablePhantomPool = isStablePhantom(pool.value.poolType);
+    const daysFromPoolCreation = differenceInDays(
```

Comment:
Rather than modifying this function would be better to have a `usePoolSnapshotSinceCreationQuery` that calls this function with the days from pool creation. So that we can still re-use this function in the future with an arbitrary number of days.

Reply:
I made it in different way. now function use days if they are provided in args, if not - days from pool creation. 
so we still can reuse this function in the future